airbyte-integrations/connectors/source-datascope/source_datascope/spec.json

```diff
@@ -0,0 +1,32 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Datascope Spec",
+    "type": "object",
+    "required": ["api_key", "form_id"],
```

Comment:
start date should be required it looks like?

Reply:
yes, going to update that.